ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -282,6 +282,47 @@ FAN_SS_DRSM8X:
   - SS
   - DRSM8X
 
+FAN_SS_DPM_ZHM_ZTM:
+  description: "Fan with sensors (temperature, humidity and low pressure)."
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  - DPM
+  - ZHM
+  - ZTM
+  opt_uses:
+  - control_mode
+  - low_differential_pressure_alarm
+  - lost_power_alarm
+  - control_panel_mode
+
+FAN_BRS:
```

Comment:
please follow convention. should be FAN_SS, but this already exists.

Reply:
Changed to FAN_SS_AL